index.js

```diff
@@ -205,7 +211,11 @@ var TruffleContractSchema = {
     // update schema version
     normalized.schemaVersion = pkgVersion;
 
-    return normalized;
+    if (options.validate) {
```

Comment:
> This is a nice feature. Are you using it anywhere outside of truffle-contract-schema?

Not yet - just figured it'd be a good option to have. I tried running various tests in the suite with `validate` turned on, but there were errors with different parts of Truffle producing barely invalid ABIs